addons/binding/org.openhab.binding.amazondashbutton/src/main/java/org/openhab/binding/amazondashbutton/internal/discovery/AmazonDashButtonDiscoveryService.java

```diff
@@ -32,8 +34,6 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import com.google.common.collect.Sets;
```

Comment:
Should / could we remove it from the manifest?

Reply:
Unfortunately not ... yet ...

https://github.com/openhab/openhab2-addons/blob/533e5f01be4d5395d9dc99b5e0fb3f22410c49f3/addons/binding/org.openhab.binding.amazondashbutton/src/main/java/org/openhab/binding/amazondashbutton/internal/pcap/PcapNetworkInterfaceService.java#L23-L26
https://github.com/openhab/openhab2-addons/blob/533e5f01be4d5395d9dc99b5e0fb3f22410c49f3/addons/binding/org.openhab.binding.amazondashbutton/src/main/java/org/openhab/binding/amazondashbutton/internal/pcap/PcapUtil.java#L14